flink-queryable-state/flink-queryable-state-runtime/src/test/java/org/apache/flink/queryablestate/itcases/AbstractQueryableStateTestBase.java

```diff
@@ -246,19 +247,20 @@ public Integer getKey(Tuple2<Integer, Long> value) {
                         .get(deadline.timeLeft().toMillis(), TimeUnit.MILLISECONDS);
             }
 
-            assertTrue("Not all keys are non-zero", allNonZero);
+            assertThat(allNonZero).isTrue().withFailMessage("Not all keys are non-zero");
 
             // All should be non-zero
             for (int i = 0; i < numKeys; i++) {
                 long count = counts.get(i);
-                assertTrue("Count at position " + i + " is " + count, count > 0);
+                assertThat(count).isGreaterThan(0).withFailMessage("Count at position " + i + " is " + count);
             }
         }
     }
 
     /** Tests that duplicate query registrations fail the job at the JobManager. */
-    @Test(timeout = 60_000)
-    public void testDuplicateRegistrationFailsJob() throws Exception {
+    @Test
+    @Timeout(60_000)
```

Comment:
We should rely on global timeout of AZP, local timeout is unstable.

Reply:
Removed @timeout.